app/models/application_record.rb

```diff
@@ -95,7 +94,7 @@ def self.update_or_create!(attributes)
     obj
   end
 
-  def self.whitelist_attributes(a_hash, options={})
+  def self.whitelist_attributes(a_hash, options = {})
```

Comment:
Side comment: maybe in the future we should consider this well-knoen discussion https://github.com/rails/rails/issues/33677